common/src/constants.rs

```diff
@@ -5,3 +5,6 @@ pub const DEV_CHAIN_ID: u64 = 31337;
 
 /// The first four bytes of the call data for a function call specifies the function to be called.
 pub const SELECTOR_LEN: usize = 4;
+
+/// Maximum size in bytes that a contract can have.
+pub const CONTRACT_MAX_SIZE: usize = 0x6000;
```

Comment:
since this is a usize, I'd prefer the actual number here and but 0x6000 in the comment